src/nomad_material_processing/vapor_deposition/__init__.py

```diff
@@ -62,6 +64,164 @@
 m_package = Package(name='Vapor Deposition')
 
 
+class InsertReduction(ArchiveSection):
```

Comment:
Another option for this would be to just nest another substrate holder inside. Might cover a bit more general cases. But this is also fine for now.

Reply:
oh, I see. Let's modify it in teh future when we face this case